network/src/peer_manager/mod.rs

```diff
@@ -762,7 +762,7 @@ where
                         );
                     }
                 } else {
-                    unreachable!(
+                    warn!(
```

Comment:
I don't even know if this is worthy of a log level of anything more than debug... Considering that this could be a means for an attacker to spam our fullnodes and burn through precious logging traffic.

Reply:
Good point! Done, thanks 😄